packages/components/demo/example/example.component.scss

```diff
@@ -0,0 +1,10 @@
+.example-container {
+  width: 80px;
+  &[data-variant="red_square"] {
+    background-color: red;
```

Comment:
nit(blocking): could you update to show how to access colour variables instead of hardcoding?